axelrod/strategies/titfortat.py

```diff
@@ -273,3 +273,59 @@ def reset(self):
         Player.reset(self)
         self.randomness_counter = 0
         self.deadlock_counter = 0
+
+
+class Gradual(Player):
+    """
+    A player that punishes defections with a growing number of defections
+    but after punishing enters a calming state and cooperates no matter what
+    the opponent does for two rounds.
+
+    http://perso.uclouvain.be/vincent.blondel/workshops/2003/beaufils.pdf """
+
+    name = "Gradual"
+    classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': False,
+        'makes_use_of': set(),
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    calming = False
+    punishing = False
+    punishment_count = 0
+    punishment_limit = 0
+
+    def strategy(self, opponent):
+
+        if self.calming:
+            self.calming = False
+            return 'C'
```

Comment:
Let's use `C` and `D` from `Axelrod.Actions` instead of explicit `'C'` and `'D'` (should already be imported at the top of the file). Same for the lines just below.